.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: .NET Core CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: .NET on ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
```

Comment:
What SDK version does this use? Latest? LTS?

Reply:
Not actually sure what it decides to use, but looking at one of my projects, it pulled 3.1.202.

Reply:
Sorry for the wrong place! How can download_depot be used these days? What addition flag is needed? Any news about that?